### ISSUE TITLE
Error after installing version 3.1.1 (exception in getForecastDataHourlyJSON [TypeError: Assignment to constant variable.])

### ISSUE THREAD

**Issue by latzi0710** [rg-engineering/ioBroker.daswetter#130]
**Describe the bug**  

After adapter is activated by cron and collecting data following error message occures:

```
2022-03-20 05:33:41.562  - info: host.ioBroker-slave instance scheduled system.adapter.daswetter.0 */15 * * * *
2022-03-20 05:33:41.566  - info: host.ioBroker-slave instance system.adapter.daswetter.0 started with pid 167043
2022-03-20 05:33:43.157  - info:  daswetter.0 (167043) starting. Version 3.1.1 in  /opt/iobroker/node_modules/iobroker.daswetter, node: v14.19.1,  js-controller: 4.0.21
2022-03-20 05:34:36.437  - info: daswetter.0 (167043) still wrong data structure from server received! repaired...
2022-03-20 05:34:36.439  - error: daswetter.0 (167043) exception in getForecastDataHourlyJSON [TypeError: Assignment to constant variable.]
2022-03-20 05:34:36.440  - info: daswetter.0 (167043) Terminated (ADAPTER_REQUESTED_TERMINATION): All data handled, adapter stopped until next scheduled moment
2022-03-20 05:34:36.979  - info: daswetter.0 (167043) cleaned everything up...
2022-03-20 05:34:37.196  - info: host.ioBroker-slave instance system.adapter.daswetter.0 terminated with code 11 (ADAPTER_REQUESTED_TERMINATION)
```

**Versions:**  
 - Adapter version: 3.1.1
 - JS-Controller version: 4.0.21
 - Node version: 14.19.1 /npm 6.14.16
 - Operating system: debian 11

**Comment by rg-engineering** [comment]
should be fixed with 3.1.2

**Comment by latzi0710** [comment]
Lokk´s good wirh 3.1.2 now 👍
